### ISSUE TITLE
URL/LINK

### ISSUE THREAD

**Issue by profesionalleet1337** [CyberTitus/Anonymous-SMS#2]
The tool is work fine, but its impossible to put url, link in him. Like many of the others gits, this doesn't allow to do its. Could be upgraded , fix somehow ? The main purpose of this type of tools, it to could be put links. Please , improve mate ! I Will be really grateful if you could done this for most people.
Happy Holidays and wish all the best !

**Comment by AbdulRKB** [comment]
Hello there, To make this tool I have used an external API service called text belt, that company has not allowed the use of URLs in the messages unless you are whitelisted because it can be abused and misused for purposes like scamming. I have made this tool with the intention of pranking.

Message from textbelt:
_Due to abuse, sending text messages containing URLs requires whitelisted permissions (send an email to support to get URL sending enabled on your account, or go to https://textbelt.com/whitelist?key=yourkey)._

**Comment by profesionalleet1337** [comment]
@CyberTitus CyberTitusHappy New Year, mate! Thank you for your feedback and guidance!
  Could you explain to me exactly what I need to do in textbelt to get this access to be able to send a message? I can't even register on their site? Why so?  As far as I understand, I don't have to pay, right? Sorry to ask you for the second time, but I really need some "pattern". Stay healthy and keep growing. If you have a discord, I will be happy to make contact with you where we can possibly discuss! Greetings !

**Comment by AbdulRKB** [comment]
To use text belt, you have to pay money and depending on how many requests you send, it costs you more money. Please give your discord ID so I can contact you. Text belt doesn't ask you to login to an account, you have to go to the "Create API Key" page where you select how many messages you want to send and the location of the numbers. After you have the API key you can send messages and to send a URL in SMS, you have to go to the page [https://textbelt.com/whitelist?key=yourkey](https://textbelt.com/whitelist?key=yourkey
) and give reasons of why you're sending the SMS and you have to verify by giving name of your domain (if it's a company).

Here is their price for **Europe**:
_$5/30 texts
$8/50 texts
$15/150 texts
$30/325 texts
$60/700 texts_

**Comment by AbdulRKB** [comment]
I have used their demo API Key which was free, that's why there is a limit on how much messages you can send to a number per day with this tool.

**Comment by profesionalleet1337** [comment]
@CyberTitus Thanks you very  much for completely detailed feedback and fast response ! I appreciate it! This is my discord: ivan.ru#1031 ( Write me message, so i could know, who are you ) .

**Comment by profesionalleet1337** [comment]
Hello, again my friend ! Would u give me your telegram or discord, where i could contact with you : ) ? 
@CyberTitus

**Comment by AbdulRKB** [comment]
What do you want to discuss

**Comment by profesionalleet1337** [comment]
> What do you want to discuss

I want to have contact with you, if its possible.

**Comment by AbdulRKB** [comment]
Any detail on what you want to contact me for?

**Comment by profesionalleet1337** [comment]
> Any detail on what you want to contact me for?

Tried with today's date, hadn't used it in a long time - get this : [-] Couldn't Send Message! [-] Test texts are temporarily disabled. Any ideas ? The software not working anymore or ?

**Comment by profesionalleet1337** [comment]
?

**Comment by AbdulRKB** [comment]
The tool used a third-party service called TextBelt for sending the SMS. The company has temporarily disabled the free use of the service and you will have to pay to them to send the SMS. Maybe in few months they will re-enable it.

**Comment by profesionalleet1337** [comment]
> The tool used a third-party service called TextBelt for sending the SMS. The company has temporarily disabled the free use of the service and you will have to pay to them to send the SMS. Maybe in few months they will re-enable it.

Thank you for your feedback and response, my friend! When do you think this will happen, and if at all, they will fix it or are you just hoping?
- Separately, are there any alternative tools based on this principle , that could do some work as well as your, just something that work still ?
- > On your site, on the projects menu I see the following : "Anonymous SMS sender " SPALK , with an actual date on it, 29/08/23, but is it just as a picture ? Is there access, to this tool, does it actually work, since, everything in your section, is as pictures ? Look's very intresting and clean .
I had asked you about discord before to keep some contact  with you , but you didn't told me  if you use it or not :) ?

**Comment by AbdulRKB** [comment]
I have no idea when they will reopen their free SMS service, as I am not affiliated with them in any way. There are many SMS tools on the web, but many of them are fake and unreliable. I don't trust them because they may log your messages in their databases.

I have used one service that claimed to be 'secure and private,' but it was later discovered that they were logging all messages along with IP addresses in their databases. The '**SPALK**' project I mentioned was just a test project I created. It is currently archived, and I have no intentions of making any changes to it anytime soon.

I tried to contact you on Discord, but I didn't receive much of a response to my messages.
